src/Login.js

```diff
@@ -0,0 +1,40 @@
+import React, {useState} from 'react';
+import { Box, Form, Button } from 'react-bulma-components';
+import Cookies from 'universal-cookie';
+
+const Login = ({usernameExists}) => {
+    const [username, setUsername] = useState('Enter a username');
+    const cookies = new Cookies();
+    const showHideLogin = usernameExists ? "hidden" : "block";
+
+    return (
+        <Box display={showHideLogin} style={{ width: 400, margin: 'auto' }}>
+            <form>
+                <Form.Field>
+                    <Form.Label>Username</Form.Label>
+                    <Form.Control>
+                        <Form.Input
+                            color="success"
+                            value={username}
```

Comment:
for this default value for the input, maybe use placeholder?  This is from 

https://bulma.io/documentation/form/input/

![image](https://user-images.githubusercontent.com/448399/128631100-66f6f85a-fc1f-4d16-b2da-897d7c09e61d.png)

Reply:
@JCTalbott @dinushkaherath

Reply:
got it used placeholder instead of set state